app/src/main/java/com/ttbmp/cinehub/app/repository/hall/HallRepository.java

```diff
@@ -12,7 +12,7 @@
  */
 public interface HallRepository {
 
-    List<Hall> getHallList(Cinema cinema);
```

Comment:
HallRepository depends now on the implementation of the Cinema entity, revert this change